docs/synapse/userguides/ug051_ing_tool.rst

```diff
@@ -0,0 +1,67 @@
+Ingest - Commandline Ingest Tool
+===============================
+
+While the Ingest subsystem in Synapse lives at synapse.lib.ingest, most users may use the standalone ingest tool
+directly.  This can be invoked with the following command: ``python -m synapse.tools.ingest <options> <ingest files>``.
+If invoked with the --help flag, the following options are listed::
```

Comment:
I would put `--help` in double backtics or replace with `-h`. it visually shows up as `-help` for me, which will not work